source/manual/covid-19-services.html.md

```diff
@@ -0,0 +1,328 @@
+- Get coronavirus support as an extremely vulnerable person
+
+There is also an interactive voice response (IVR) automated phone
+service provided by AWS.  This is for users who do not have access to
+the internet.
+
+
+## Offer coronavirus support from your business
+
+This service allows businesses to tell us how they might be able to
+help with the response to coronavirus.  This may include goods or
+services such as medical equipment, hotel rooms or childcare.
+
+- [Start page](https://www.gov.uk/coronavirus-support-from-business)
+- [Service](https://coronavirus-business-volunteers.service.gov.uk/medical-equipment)
+- [GitHub](https://github.com/alphagov/govuk-coronavirus-business-volunteer-form)
+
+
+## Get coronavirus support as an extremely vulnerable person
+
+This service allows citizens identified as vulnerable by the NHS to
+tell us if they need help accessing essential supplies and support.
+Users will have received a link to the service in a letter or a text
+message from the NHS, or been advised by their GP to fill in the form.
+They can fill the form in themselves, or someone else can fill it in
+for them.
+
+- [Start page](https://www.gov.uk/coronavirus-extremely-vulnerable)
+- [Service](https://coronavirus-vulnerable-people.service.gov.uk/live-in-england)
+- [GitHub](https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form)
+
+
+## Architecture
+
+Both applications have a similar architecture of:
+
+- **Framework:** Ruby on Rails
+- **Database:** AWS RDS (business) and DynamoDB (vulnerable people)
+- **Hosting:** PaaS
+- **CDN:** AWS CloudFront
+- **CI:** Travis
+- **CD:** Concourse
+- **Logging:** Sentry and Logit
+
+### Application structure
+
+Each application is a standard Rails application with:
+
+- question pages, each has their own controller, view and route
+- a check your answers page
+- a confirmation ("Thank you") page
+- ineligible pages (vulnerable people form only)
+- a privacy page
+
+To run one of the applications locally, see the README in the GitHub repo.
+
+
+## Session and data storage
+
+### Session storage
+
+While the user is filling out the form, we use session storage to
+store the user's data.
+
+For the vulnerable people form, we store session data in an encrypted
+cookie, persisted in the browser and sent back to the server for every
+question.  Cookies have a limit of 4KB, so this approach could cause
+errors if the user submits large inputs.
+
+For the business volunteering form, we store the following for 2
+hours:
+
+- the session id in an encrypted cookie
+- the session data in Redis
+
+### Data storage
+
+For both applications, when the user submits the form on the "Check
+your answers" page, the application writes user data to the database.
+The application only has permission to write items to the database.
+For security and privacy reasons, there's no way to read or change
+already-submitted user data.
+
+Developers should not have access to the production database.
```

Comment:
Same as my previous comments, only for the vulnerable people form.  The business volunteering database is being treated like any other GOV.UK personal data store (e.g. signon or email-alert-api).

Reply:
[New commit here](https://github.com/alphagov/govuk-developer-docs/pull/2423/commits/465f1c6a1ac4306940069b076127083db843e7fd)